ecosystem/indexer/migrations/2022-08-02-204644_token_propertys/up.sql

```diff
@@ -0,0 +1,14 @@
+-- Your SQL goes here
+CREATE TABLE token_propertys
```

Comment:
Ideally rename to token_properties.

Reply:
I tried to name it token_properties and the diesel fail to recognize the name.  Diesel sucks and unfortunately I cannot help.